site-vars.yaml

```diff
@@ -0,0 +1,6 @@
+# Placeholder to update site specific variables
+
+# Internal docker pull-through cache/mirror registry
+zuul_site_mirror_fqdn: 10.245.167.88
+docker_insecure_registries:
+  - 10.245.167.88:8082
```

Comment:
I can see these now as defined, but where are they used?

Reply:
@ajkavanagh role publish-charm includes role ensure-docker [1] and ensure-docker role includes role use-docker-mirror [2] and [3] [4] are places where those vars are used.

[1] https://opendev.org/zuul/zuul-jobs/src/branch/master/roles/use-docker-mirror/tasks/main.yaml#L6
[2] https://opendev.org/zuul/zuul-jobs/src/branch/master/roles/ensure-docker/tasks/docker-upstream.yaml#L12
[3] https://opendev.org/zuul/zuul-jobs/src/branch/master/roles/use-docker-mirror/tasks/main.yaml#L6
[4] https://opendev.org/zuul/zuul-jobs/src/branch/master/roles/use-docker-mirror/templates/daemon.json.j2